cmd/command_test.go

```diff
@@ -0,0 +1,39 @@
+package cmd
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/spf13/cobra"
+)
+
+func executeCommand(root *cobra.Command, args ...string) (output string, err error) {
+	_, output, err = executeCommandC(root, args...)
+	return output, err
+}
+
+func executeCommandC(root *cobra.Command, args ...string) (c *cobra.Command, output string, err error) {
+	buf := new(bytes.Buffer)
+	root.SetOutput(buf)
+	root.SetArgs(args)
+	c, err = root.ExecuteC()
+	return c, buf.String(), err
+}
+
+func TestCommandReturnErrorWhenServerNotRunning(t *testing.T) {
+	tests := []struct {
+		command *cobra.Command
+		name    string
+	}{
+		{statusCmd, "status"},
+		{statsCmd, "stats"},
+	}
+
+	for _, tc := range tests {
+		RootCmd.AddCommand(tc.command)
```

Comment:
Would it be a good idea to defer `RootCmd.RemoveCommand()` after this, to ensure it doesn't pollute the global `RootCmd` for subsequent tests?

Reply:
Not a big deal, but nice to have. Fixed.